modules/update-vsphere-virtual-hardware-on-template.adoc

```diff
@@ -22,6 +22,11 @@ Once converted from a template, do not power on the virtual machine.
 ====
 
 . Update the virtual machine (VM) in the {vmw-full} client. Complete the steps outlined in link:https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.vm_admin.doc/GUID-60768C2F-72E1-42E0-8A17-CA76849F2950.html[Upgrade the Compatibility of a Virtual Machine Manually] ({vmw-full} documentation).
++
+[IMPORTANT]
```

Comment:
Not a merge blocker and out-of-scope for this PR:
Each of steps in this procedure is followed by and admonition. While these admonitions aren't technically "stacked", one after another, it does feel a bit overwhelming and distracting. Consider refactoring the procedure to find other ways to highlight this critical information to the customer.

Reply:
@michaelryanpeter nice call out! This was my last OCP ticket since I moved to RHOAI and I'm a little swamped, but if you're okay creating a Jira for this, I think it's a great idea!